packages/trip-form/src/DropdownSelector/index.tsx

```diff
@@ -67,6 +71,23 @@ export default function DropdownSelector({
   );
 
   const id = `id-query-param-${name}`;
+  const transformedOptions: TransformedOption[] = options.map(option => ({
+    label: option.text,
+    value: option.value
+  }));
+
+  const customStyles = {
```

Comment:
There is no hover effect on the options, which makes it very difficult to see which option is being potentially selected. The new dropdowns are also larger than the old ones, which makes for some very awkward formatting in `TripOptions`.

Reply:
No focus styling also means that this is next to impossible for keyboard users to use.